src/spec-common/injectHeadless.ts

```diff
@@ -535,12 +535,13 @@ async function runLifecycleCommand({ lifecycleHook }: ResolverParameters, contai
 			if (err && (err.code === 130 || err.signal === 2)) { // SIGINT seen on darwin as code === 130, would also make sense as signal === 2.
 				infoOutput.raw(`\r\n\x1b[1m${lifecycleHookName} interrupted.\x1b[0m\r\n\r\n`);
 			} else {
+				infoOutput.write(toErrorText(`${err.cmdOutput}`));
```

Comment:
Should we check if `err.cmdOutput` exists before we output? We should try to avoid null refs.

Reply:
done